encoding/testdata/jsonc/JSONTestSuite/README.md

```diff
@@ -0,0 +1,35 @@
+This directory contains the test cases copied from
+[JSONTestSuite](https://github.com/nst/JSONTestSuite) organization.
+
+The files in `./test_parsing/` have been copied from
+https://github.com/nst/JSONTestSuite/tree/d64aefb55228d9584d3e5b2433f720ea8fd00c82/test_parsing.
+The files in `./test_transform/` have been copied from
+https://github.com/nst/JSONTestSuite/tree/d64aefb55228d9584d3e5b2433f720ea8fd00c82/test_transform.
+
+The license of the copied JSON file is as follows.
+
+https://github.com/nst/JSONTestSuite/blob/d64aefb55228d9584d3e5b2433f720ea8fd00c82/LICENSE
+
+```
+MIT License
+
```

Comment:
I have included the license in the Readme since comments are not allowed in the JSON file.  Would this be ok?

Reply:
Thanks! Looks good